salt_stack/salt/lambda-dev/files/makebaseenv

```diff
@@ -1,21 +0,0 @@
-#! /usr/bin/env bash
```

Comment:
Deleted this file because its functionality is incorporated into `lambda-dev/init.sls`.